internal/installs/installs_test.go

```diff
@@ -48,35 +48,67 @@ func TestInstallPath(t *testing.T) {
 }
 
 func TestInstalled(t *testing.T) {
-	conf, plugin := generateConfig(t)
-
 	t.Run("returns empty slice for newly installed plugin", func(t *testing.T) {
+		conf, plugin := generateConfig(t)
 		installedVersions, err := Installed(conf, plugin)
 		assert.Nil(t, err)
 		assert.Empty(t, installedVersions)
 	})
 
 	t.Run("returns slice of all installed versions for a tool", func(t *testing.T) {
+		conf, plugin := generateConfig(t)
 		mockInstall(t, conf, plugin, "1.0.0")
 
 		installedVersions, err := Installed(conf, plugin)
 		assert.Nil(t, err)
 		assert.Equal(t, installedVersions, []string{"1.0.0"})
 	})
+
+	t.Run("returns installed versions including symlinks", func(t *testing.T) {
+		conf, plugin := generateConfig(t)
+		mockInstall(t, conf, plugin, "1.0.0")
+		aliasVersion(t, conf, plugin, "1.0.0", "latest")
```

Comment:
We do not support aliases in asdf core. Please rename this function to something that indicates it only generates a symlink, and please remove the `latest` version as `latest` is never permitted in `.tool-versions` files.